CONTRIBUTING.md

```diff
@@ -79,6 +79,30 @@ scripts/format.sh
 5. You may merge the Pull Request in once you have the sign-off of another developers, or if you
    do not have permission to do that, you may request the reviewer to merge it for you.
 
+
+### Pull Request previews
+
+When a PR includes a docs change, a preview can be automatically deployed to Netlify.
+For non-fork PRs, this will "just work," but for fork PRs, an authorized user must first
+check the PR for dangerous changes, then post a comment in the PR starting with
+`/deploy-docs-preview` to trigger the deployment. Review the workflow configuration to
+understand why.
+
+Configuration is done following [this
+guide](https://mfisher87.github.io/posts/website-repo-pr-preview/) using an individual's
+GitHub account (TODO: Whose?). If something goes wrong and this person can't be
```

Comment:
Should we use my work GitHub account for this? Or @betolink 's?

Reply:
I'd say yours is fine, does it need an individual, can it be configured for the repo?

Reply:
Unfortunately has to be an individual. But since we plan to fix ReadTheDocs tomorrow, I expect we will want to decline this PR.

Reply:
>Unfortunately has to be an individual

Should have explained: team features cost money! :money_with_wings: